README.md

```
@@ -39,7 +39,782 @@ For information on reporting security vulnerabilities in N|Solid, see
 
 ## Contributing to N|Solid
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)